src/table/TableHeadWrapper.jsx

```diff
@@ -85,7 +87,6 @@ function TableHeadWrapper({
                 {column.label}
                 {isFocusInHead && (
```

Comment:
we also could have the check for isCurrentColumnActive in order to announce the "Press Space ..." thing only on actively sorted columns? what do you think? is it a good idea or not?